pkg/controller/account/ec2.go

```diff
@@ -186,6 +187,46 @@ func (r *ReconcileAccount) InitializeRegion(
 	return nil
 }
 
+// GetSubnetID builds the vpc and create subnet
+func GetSubnetID(reqLogger logr.Logger, client awsclient.Client) (string, error) {
+	// TODO: wait time for vpc to create?
+	vpcID, err := CreateVpc(reqLogger, client)
+	if err != nil {
+		controllerutils.LogAwsError(reqLogger, "Unable to create VPC", nil, err)
+	}
+	subnetID, err := CreateSubnet(reqLogger, client, "10.0.1.0/24", vpcID)
```

Comment:
If you're only creating 1 subnet for 1 VPC, making the CIDR ranges match likely makes the most sense. Then extracting the value to a single var in the package makes it easier to change to adjust the value too

Reply:
Fixed